.github/workflows/formatting-check.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           ${{ runner.os }}-maven-
 
     - name: Build configuration
-      run: mvn clean install -pl :debezium-ide-configs
+      run: mvn clean install -pl :debezium-ide-configs,:debezium-checkstyle
```

Comment:
So thinking about it, I think this is what should be run here:

    mvn clean package -DskipTests=true -DskipITs=true process-sources -Dformat.formatter.goal=validate -Dformat.imports.goal=check

Reply:
This also means Checkstyle, Impsort and Format check can be skipped in the main test run which save some time there.